setup.py

```diff
@@ -103,11 +104,15 @@ def has_ext_modules(self) -> bool:
         return enable_aot
 
 
+bdist_wheel_options = {}
+if not sysconfig.get_config_var("Py_GIL_DISABLED"):
```

Comment:
For AOT, can we use an environment variable to control the behavior here? e.g. `FLASHINFER_AOT_USE_PY_LIMITED_API`, and defaults it to 1.

The reason is we always want to compile wheels for users w/ Py_LIMITED_API on CI/CD. But user might also use this script to compile flashinfer by their own, it would be great if we can control the behavior in AOT.

Reply:
> Would https://github.com/flashinfer-ai/flashinfer/commit/29a59a6be3b5b34fffd0a6b1fe94788c6c7f6d7c work?

Yes it looks good to me, thanks for the update.

> This doesn’t affect the current CI/CD anyway, since it only builds wheels for 3.10 and 3.12 as far as I can tell.

There is a public CI/CD which targets multi-python users.